XamarinCommunityToolkitSample/Pages/Behaviors/EventToCommandBehaviorPage.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:vm="clr-namespace:XamarinCommunityToolkitSample.ViewModels.Behaviors"
+             xmlns:behaviors="clr-namespace:XamarinCommunityToolkit.Behaviors;assembly=XamarinCommunityToolkit"
+             x:Class="XamarinCommunityToolkitSample.Pages.Behaviors.EventToCommandBehaviorPage">
+
+    <ContentPage.BindingContext>
+        <vm:EventToCommandBehaviorViewModel />
+    </ContentPage.BindingContext>
+
+    <ContentView Padding="{StaticResource ContentPadding}">
+        <StackLayout VerticalOptions="CenterAndExpand"
+                     Spacing="20">
+            <Label Text="{Binding ClickCount, StringFormat='{0} clicks'}" />
+            <Button Text="CLICK ME"
+                    TextColor="White"
+                    BackgroundColor="{StaticResource PrimaryColor}">
+                <Button.Behaviors>
+                    <behaviors:EventToCommandBehavior
+                        EventName="Clicked"
+                        Command="{Binding Command}" />
```

Comment:
Maybe rename the actual command to something like `IncrementCommand` so it's clear that we are not referring to the `Command` property in this case